src/entrypoints/main/router/pages/community-topic-page/components/post/styled.ts

```diff
@@ -0,0 +1,40 @@
+import styled, { css } from 'styled-components';
+
+import { theme, Colour } from '../../../../../app/theme';
+
+type postProps = {
+  $firstPost: boolean;
+};
+
+const Post = styled.div<postProps>`
+  display: flex;
+  flex-direction: column;
+  font-size: ${theme.fontSize('FS10')};
+  color: ${theme.colour(Colour.BLUE, 'B52')};
+  padding: ${theme.spacing('S24')};
+  background: ${theme.colour(Colour.NEUTRAL, 'N02')};
+  margin-bottom: ${theme.spacing('S10')};
+
+  ${({ $firstPost }) =>
```

Comment:
Alternativ løsning er at denne komponenten ikke bryr seg om dette, denne komponenten blir renere. Ansvaret kan ligge hos parent-komponent. Ser for meg at styling basert på første, siste osv. er veldig kontekstavhengig. Da kan du heller i parent-komponent bruke first-child osv for å gjøre slike tilpasninger.